pandas/core/series.py

```diff
@@ -2658,6 +2661,193 @@ def fillna(self, value=None, method=None, axis=None, inplace=False,
                                           limit=limit, downcast=downcast,
                                           **kwargs)
 
+    def replace(self, to_replace=None, value=None, inplace=False, limit=None,
```

Comment:
can we share *most* / all of this via ``_shared_docs``? otherwise end up with lots of duplication here